README.md

```diff
@@ -12,34 +12,32 @@ Bozza ver. 01.047 – 2020-04-12 &ndash; Scritto con [***Mark Text***](https://m
 
 Ci trovate anche **in Diaspora, rete sociale decentralizzata:**
 
-- Link all'account, che contiene pubblicazioni in relazione a questo manifesto:
+- Link all'account che contiene pubblicazioni in relazione a questo manifesto:
   
   https://mondiaspora.net/people/14281b90538e0138a38a03f7e1ed1bc8
 
 - Link diretto alla breve presentazione di questo manifesto:
   
   https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653
   
-  Non abbiamo per ora un nostro sito web con base dati, onde proporre una raccolta di firme, una sorta di petizione.  
-  **Commentare [quella pubblicazione](https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653) in Diaspora è al momento un modo per segnalarci la vostra approvazoine.** Se e quando vorrete farlo, potrete scrivere qualche cosa come:  
+  Ancora non abbiamo predisposto un nostro sito web con base dati, dove proporre una raccolta di firme o petizione di sorta.  
+  **Commentare [quella pubblicazione](https://mondiaspora.net/posts/ab05452057880138212d0e36a79e7653) in Diaspora è al momento il modo di supportare il nostro lavoro e segnalarci la vostra approvazoine.** Se vorrete farlo, ci potrete scrivere qualche cosa come:  
   **"Approvo il manifesto *Nuovo Paradigma 1.0* (bozza ver. nn.nnn)"**.  
-  La versione che vi è piaciuta potrete anche copiarvela ovviamente (così come qualunque altra).  
-  **Se avete una "facciata" nel web, potrete segnalare nella stessa il manifesto.**  
-  Spendere 5 o 10 minuti del vostro tempo per aprire un acount gratuito su Diaspora sarà l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
+  La versione che più vi è piaciuta potrete anche copiarla (e lo stesso dicasi per qualunque altra).  
+  **Se disponete di account, nick o una qualunque "facciata web" potrete segnalare l'esitenza del manifesto tramite la stessa.**  
+  Vi ringrazieremo se spenderete 5-10 minuti del vostro tempo per aprire un account gratuito su Diaspora e sarà anche l'occasione di uscire un po' dalle grinfie di Facebook, con la sua censura e la sua connivenza con *Big Data*.
 
 &nbsp;
 
-Almeno durante un primo periodo, questo sarà un lavoro in evoluzione, anche se vorremo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione (potremmo valutare se effettuare delle *release*).
+Almeno per un primo periodo questo manifesto é da cosiderare come un lavoro in evoluzione, anche se vorremmo marcare il raggiungimento di certe tappe di sviluppo del pensiero e dell'azione entro breve tempo (potremmo valutare se effettuare delle *release*).
```

Comment:
"è da considerare" 'è' andrebbe con accento grave, non 'é'
"vorremo" l'avevo messo all'indicativo, non era un errore di stompa, cioè quel traguardo _ce lo proponiamo_, mentre vorremmo è più "modesto", "ci piacerebbe".... È comunque un'opzione possibile (anche se preferirei evitare la consonanza con potremmo perché non suoni pesante, ma è una considerazione del tutto marginale).

Dimmi che te ne pare.

Reply:
Si, vedo che mi continua a piazzare la e con accento sbagliato 😂 Capiterà ancora, vedrai e correggimela pure perché spesso digito veloce e non mi ricordo di andarla a cambiare

Reply:
^ nooo please, risolvilo, già io ho fatto degli orari pazzeschi e mi sono scappati degli errori, se tu ne aggiungi non accidentalmente ma sistematicamente...

tra l'altro stavo pensando che forse dovremmo dare una sgrossata di massima, CIOÈ ragionare in termini MACROSCOPICI, guadagnare una visione d'insieme del documento, della struttura, dei contenuti... INUTILE SPACCARE IL CAPELLO IN QUATTRO CON LA SCORREVOLEZZA, PER ORA, SE POI CI RENDIAMO CONTO CHE CERTE PARTI ANDREBBERO TOLTE O L'ORDINE CAMBIATO O CERTI ARGOMENTI GIUSTIFICATI IN ALTRO MODO... no?